graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -174,14 +172,18 @@ def test_has_substring_precedence(self, backend_name):
         self.assertResultsEqual(graphql_query, parameters, backend_name, expected_results)
 
     @integration_fixtures
-    def test_get_graphql_schema_from_orientdb_schema(self):
-        schema_records = self.graph_client.command(ORIENTDB_SCHEMA_RECORDS_QUERY)
-        schema_data = [x.oRecordData for x in schema_records]
-        type_overrides = {
-            "UniquelyIdentifiable": {"uuid": GraphQLID},
-            "Animal": {"birthday": GraphQLDate},
-            "Event": {"event_date": GraphQLDateTime}
-        }
-        schema, _ = get_graphql_schema_from_orientdb_schema_data(schema_data, type_overrides)
+    def test_snapshot_graphql_schema_from_orientdb_schema(self):
+        schema, _ = generate_schema(self.graph_client)
         compare_ignoring_whitespace(self, SCHEMA_TEXT, print_schema(schema), None)
+
+    @integration_fixtures
+    def test_override_field_types(self):
```

Comment:
Add a comment explaining how it's being done. It's not obvious that this implementation does what the name of the test suggests.